qt/python/mantidqt/dialogs/spectraselectordialog.py

```diff
@@ -325,6 +329,11 @@ def _parse_wksp_indices(self):
             selection.wksp_indices = wksp_indices
             selection.plot_type = self._ui.plotType.currentIndex()
 
+            if self._errorsset:
+                selection.errors = True
+            else:
+                selection.errors = False
```

Comment:
This could be simplified to `selection.errors = self._errorsset`

Reply:
Not sure how I managed to overcomplicate that, thanks!